docs/search/current_queries/works.md

```diff
@@ -275,3 +165,23 @@ we want to prioritise the third result. Based on user feedback documented here <
   }
 }
 ```
+
+### Users should see documents which contain exact matches to their query above documents which contain partial matches
+
+We've introduced a section of the query which is designed to match documents which contain more exact matches to the user's query, including casing and some punctuation, multi-matched over a few key title fields. This field is analysed with a `shingle` filter, meaning spans of multiple matched tokens will score even more highly. The field mappings also use the discard the lowercase filter and include a `hyphens` char filter, meaning that hyphenated words will be treated as a single token. This is important because we want to match `x-ray` over `x ray` when someone searches for `x-ray`, or `AIDS` over `aids` when someone searches for `AIDS`.
+
+```json
+{
+    "multi_match": {
+        "query": "{{query}}",
+        "fields": [
+            "query.title.shingles_cased^1000.0",
+            "query.alternativeTitles.shingles_cased^100.0",
+            "query.partOf.title.shingles_cased^10.0"
+        ],
+        "type": "most_fields",
+        "operator": "And",
+        "_name": "shingles_cased"
+    }
+}
+```
```

Comment:
Not sure if this was here on the first pass, but this is really nice (and potentially solves the problem we discussed about comments in the mapping/query).

This feels like it could be edited into a document we could share with non-data scientists, e.g. Collections or me – each section starts with a digestible summary and a few examples, and then you hide the "how it works" behind a collapsible `<details>` block.

That's not a "now" problem, but more a thought for the future.